app/src/main/java/com/sun/wsm_android_v2/screen/main/MainActivity.kt

```diff
@@ -5,13 +5,15 @@ import androidx.databinding.DataBindingUtil
 import com.sun.wsm_android_v2.R
 import com.sun.wsm_android_v2.base.BaseActivity
 import com.sun.wsm_android_v2.databinding.ActivityMainBinding
+import com.sun.wsm_android_v2.screen.login.LoginFragment
 import com.sun.wsm_android_v2.screen.timesheet.TimeSheetFragment
 import com.sun.wsm_android_v2.util.extension.addFragmentToActivity
 
 class MainActivity : BaseActivity() {
 
     private lateinit var viewModel: MainActivityViewModel
     private val mTimeSheetFragment : TimeSheetFragment by lazy { TimeSheetFragment.newInstance() }
```

Comment:
sửa lại tên chỗ này giúp a. a bị sai convention
```suggestion
    private val timeSheetFragment : TimeSheetFragment by lazy { TimeSheetFragment.newInstance() }
```